src/Site.js

```diff
@@ -442,36 +443,38 @@ class Site {
    */
   collectAddressablePages() {
     const { pages } = this.siteConfig;
-    const addressableGlobs = pages.filter(page => page.glob);
-    this.addressablePages = pages.filter(page => page.src);
+    const pagesFromSrc = _.flatMap(pages.filter(page => page.src), page => (Array.isArray(page.src)
+      ? page.src.map(pageSrc => ({ ...page, src: pageSrc }))
+      : [page]));
```

Comment:
Thanks for the feedback and reviews! @yash-chowdhary @damithc @marvinchin 

> Yeah, this looks good to me 🚀 Supporting an array of src sounds great as well! Let's do it in another PR?

My bad, pushed this before seeing this message. Its just a three line change here ^ though, let me know if it should be in a separate PR!

Reply:
> Its just a three line change here ^ though, let me know if it should be in a separate PR!

I think it can be left in this PR. 👍